content/academy/anti_scraping/proxies.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Proxies
+description: Learn all about proxies, how they work, and how they can be leveraged in a scraper to avoid blocking and other anti-scraping tactics.
+menuWeight: 4.1
+category: courses
+paths:
+- anti-scraping/proxies
+---
+
+# [](#proxies) Proxies
```

Comment:
Generally, unless you want the anchor tag link to be different from the heading, you don't need to add it since we generate them automatically on web. Only custom anchor links need adding, e.g. `# [](#cool-proxies) Proxies`

I don't really mind if you wanna add them anyway, it just might save you time - I personally find all the brackets and dashes annoying to type :)